monai/deploy/operators/monai_seg_inference_operator.py

```diff
@@ -290,23 +294,39 @@ def verify_suffix(self, filename: Union[Sequence[str], str]) -> bool:
         return True
 
     def read(self, data: Union[Sequence[str], str], **kwargs) -> Union[Sequence[Any], Any]:
-        # Really does not have anything to do.
-        return self.input_image.asnumpy()
+        # Really does not have anything to do. Simply return the Image object
+        return self.input_image
 
     def get_data(self, input_image):
         """Extracts data array and meta data from loaded image and return them.
 
         This function returns two objects, first is numpy array of image data, second is dict of meta data.
         It constructs `affine`, `original_affine`, and `spatial_shape` and stores them in meta dict.
-        A single image is loaded with a single set of metadata as of now."""
+        A single image is loaded with a single set of metadata as of now.
```

Comment:
Suggestion for clarifying pixel-order in an Image instance

1. Use an enum to indicate the native pixel-order-format inside an Image instance. We could use an enum to represent all possible image formats:
```
 class PixelFormat(Enum):
 DHW = 1
 WHD = 2
 WDH = 3
```

2.  We can then have a method  inside Image class 

change_pixel_format (destination_pixel_format_type : PixelFormat)

This can be implemented in the next release

Reply:
Added comments to state that similar to the MONAI ITKReader, the numpy array of the image itself is of index order DHW, but the get_data() impl of the concrete classes will all need to align with MONAI NibabelReader, which return numpy array of the index order WHD(C) when its get_data() is called.